storage/client/client.go

```diff
@@ -395,7 +395,7 @@ func (c *StorageClient) Entity(ctx context.Context, r *EntityRequest) (*Entity,
 	nodeRows, err := c.db.Query(
 		ctx,
 		qf.EntityNodeIdsQuery(),
-		r.EntityID,
+		r.EntityID.String(),
```

Comment:
what's entity ID?

Reply:
It's a pubkey. Or rather, a pointer to a pubkey, though it seems it can never be `nil`, from my cursory reading of the code. IDK if it actually has to be converted to a string, but the same type is converted in several other places.